index.js

```diff
@@ -66,14 +66,14 @@ export class Dagula {
   }
 
   /**
-   * @param {import('multiformats').CID|string} cid
+   * @param {CID[]|CID|string} cid
    * @param {{ signal?: AbortSignal }} [options]
    */
   async * get (cid, options = {}) {
     cid = typeof cid === 'string' ? CID.parse(cid) : cid
     log('getting DAG %s', cid)
-    let cids = [cid]
-    while (true) {
+    let cids = Array.isArray(cid) ? cid : [cid]
+    while (cids.length > 0) {
```

Comment:
Can probably remove `if (!nextCids.length) break` later if doing the check up here...